README.md

```diff
@@ -72,16 +81,16 @@ Please refer to [native readme](native/README.md) if want to develop native appl
 
 ## Example Project
 
-- [Example Cases](https://github.com/cocos/cocos-example-projects): Simple yet expressive demo scenes for baseline testing and topic-specific case study.
 - [Mind Your Step 3D](https://github.com/cocos/cocos-tutorial-mind-your-step): Beginner's step-by-step tutorial project repo.
-- [UI Demo](https://github.com/cocos/cocos-example-ui): Use cases for various kinds of UI components.
 - [Test Cases](https://github.com/cocos/cocos-test-projects): Unit test scenes for every engine module.
+- [Example Cases](https://github.com/cocos/cocos-example-projects): Simple yet expressive demo scenes for baseline testing and topic-specific case study.
+- [Awesome Cocos](https://github.com/cocos/awesome-cocos): You can find out other useful tools and show cases here.
 
 ## Links
 
-- [Official site](https://www.cocos.com/products#CocosCreator)
-- [Download](https://www.cocos.com/creator)
+- [Official site](https://www.cocos.com/)
+- [Download](https://www.cocos.com/creator/download)
 - [Documentation](https://docs.cocos.com/creator/manual/en/)
 - [API References](https://docs.cocos.com/creator/api/en/)
 - [Forum](https://discuss.cocos2d-x.org/c/creator)
-- [Road Map](https://trello.com/b/JWVRRxMG/cocos-creator-roadmap)
+- [Projects and road map](https://github.com/orgs/cocos/projects?query=is%3Aopen&type=new)
```

Comment:
There's so little content in Projects and it's actually hard for users to find the key updates. So I think trello could continue to be used.

Reply:
Trello is not tracked and regularly updated, we should do the effort to use Projects, and it's been gradually adopted, it's also much more reasonable with our open issues.